DEPRECATIONS-v2.md

```diff
@@ -0,0 +1,21 @@
+# Deprecations
+
+Introducing deprecations and feature flags to be removed in the version 2 lists.
+
+💁 Please follow these steps to safely upgrade your Design System to Spirit Design System v2 components.
+
+## spirit-web v2
+
+- [Deprecations](./packages/web/DEPRECATIONS-v2.md)
+
+## spirit-web-react v2
+
+- [Deprecations](./packages/web-react/DEPRECATIONS-v2.md)
+
+## spirit-web-twig v2
+
+- _spirit-web-twig_ was released as version 2 already
```

Comment:
Maybe you should point this to the existing migration guide.

Reply:
I suppose that this file will be renamed to the `UPGRADE-v2.md` after the release of the version 2.